nextcord/__init__.py

```diff
@@ -4,7 +4,7 @@
 
 A basic wrapper for the Discord API.
 
-:copyright: (c) 2015-present Rapptz
+:copyright: (c) 2015-2021 Rapptz & 2021-present tag-epic
```

Comment:
and from 2021 on?

Reply:
Well only this one cause i am not spending another 20 minutes copy-pasting license info again

Reply:
this one here is different from the others and should definitly be changed

Reply:
yeah

> this one here is different from the others and should definitly be changed